flake.nix

```diff
@@ -33,7 +33,6 @@
               mkdir --parents "$(pwd)/tmp"
               export PATH="$(pwd)/dist/bin:$(npm root)/.bin:$PATH"
               npm install --ignore-scripts
-              npx patch-package
```

Comment:
Was this added by @amydevs and now how is patching done?

Reply:
See below comment.